android/src/main/java/com/bugsnag/BugsnagReactNative.java

```diff
@@ -27,7 +29,7 @@
     private ReactContext reactContext;
     private String libraryVersion;
     private String bugsnagAndroidVersion;
-    static final Logger logger = Logger.getLogger("bugsnag-react-native");
+    public static final Logger logger = Logger.getLogger("bugsnag-react-native");
```

Comment:
why is this change necessary?

Reply:
`JavaScriptException` needs to be in a different package to access package-private methods in `BugsnagException`, which meant it could no longer access this field.